.github/workflows/keep-supabase-active.yml

```diff
@@ -0,0 +1,15 @@
+name: Query supabase twice a week
+
+on:
+  schedule:
+    - cron: '0 0 * * 0,4' # Runs every Sunday and Thursday day at midnight
```

Comment:
"at least once every 7 days", I'm not sure if just once a week is enough

Reply:
Twice a week would be enough I imagine I think we have to wait and  monitor if it don't get inactive again.